integration/emr/alluxio-emr.sh

```diff
@@ -166,12 +166,14 @@ install_alluxio() {
   # Unpack and inflate the release tar
   # TODO logic for different compression formats, s3 URIs, git URIs, etc.
   sudo cp "${release}" /opt/
-  sudo tar -xvf "/opt/${release}" -C /opt/
+  sudo tar -xpvf "/opt/${release}" -C /opt/
```

Comment:
p?

Reply:
We chown /opt/alluxio with `alluxio:alluxio` anyways so is this even necessary?

Reply:
this is to maintain mode, specifically of the logs/user_log/ directory